packages/server/utils/stripe/StubStripeManager.ts

```diff
@@ -0,0 +1,24 @@
+import Stripe from 'stripe'
+import {toEpochSeconds} from '../epochTime'
+import StripeManager from './StripeManager'
+
+// using proxy to start gradual coverage of StripeManager
+export default function StubStripeManager() {
+  return new Proxy<StripeManager>(new StripeManager(), {
```

Comment:
+2 I see what you did here. I think this is a good way forward. At the moment we're using our test keys, but ideally we don't want to make calls to remote on CI.